test/support/live_views/general.ex

```diff
@@ -292,3 +292,10 @@ defmodule Phoenix.LiveViewTest.AssignsNotInSocketLive do
   def mount(_params, _session, socket), do: {:ok, socket}
   defp boom(socket), do: socket.assigns.boom
 end
+
+defmodule Phoenix.LiveViewTest.ErrorInMountLive do
+  use Phoenix.LiveView
+
+  def render(assigns), do: ~L|<div>I crash in mount</div>|
+  def mount(_params, _session, _socket), do: raise("boom")
```

Comment:
You can probably use the `AssignsNotInSocketLive` LV instead of adding a new one. :)

Reply:
Unfortunately the `AssignsNotInSocketLive` LV throws an exception in `render/1` instead of `mount/3` or `handle_params/3` where I needed it to trigger these telemetry events. My current solution is to make an `ErrorsLive` that takes a `"crashes_on"` query string param that allows me to trigger an exception in the following scenarios:

* `disconnected_mount`
* `connected_mount`
* `disconnected_handle_params`
* `connected_handle_params`

If there's a better way to do this, I'd be happy to make any updates.